src/opower/utilities/glendalewaterandpower.py

```diff
@@ -0,0 +1,46 @@
+"""Glendale Water and Power (GWP)."""
+
+from typing import Optional
+
+import aiohttp
+
+from ..const import USER_AGENT
+from .base import UtilityBase
+
+
+class GlendaleWaterAndPower(UtilityBase):
+    """Glendale Water and Power (GWP)."""
+
+    @staticmethod
+    def name() -> str:
+        """Distinct recognizable name of the utility."""
+        return "Glendale Water and Power (GWP)"
+
+    @staticmethod
+    def subdomain() -> str:
+        """Return the opower.com subdomain for this utility."""
+        return "gwp"
+
+    @staticmethod
+    def timezone() -> str:
+        """Return the timezone."""
+        return "America/Los_Angeles"    
+
+    @staticmethod
+    async def async_login(
+        session: aiohttp.ClientSession,
+        username: str,
+        password: str,
+        optional_mfa_secret: Optional[str],
+    ) -> None:
+        """Login to the utility website."""
+        async with session.post(
+            "https://gwp.opower.com/ei/edge/apis/user-account-control-v1/cws/v1/gwp/account/signin",
+            json={
+                "username": username,
+                "password": password,
+            },
+            headers={"User-Agent": USER_AGENT},
+            raise_for_status=False,

```

Comment:
Shouldn't this be True? We similarly need to fix BurbankWaterAndPower. If you enter incorrect credentials the code won't catch the error and continue which should fail with 401 anyway but it would be better to fail early and prevent Home Assistant from setting up the integration until valid credentials are entered. So change this to True, confirm you can still login and confirm incorrect credentials fail with InvalidAuth.

Reply:
I changed this to True. Let me know if it breaks login.